synfig-studio/src/gui/workarea.cpp

```diff
@@ -387,28 +386,15 @@ WorkArea::save_meta_data()
 	{
 		String data;
 		GuideList::const_iterator iter;
-		for(iter=get_guide_list_x().begin();iter!=get_guide_list_x().end();++iter)
-		{
-			if(!data.empty())
-				data+=' ';
-			data+=strprintf("%f",*iter);
-		}
-		if(!data.empty())
-			canvas_interface->set_meta_data("guide_x",data);
-		else if (!canvas->get_meta_data("guide_x").empty())
-			canvas_interface->erase_meta_data("guide_x");
-
-		data.clear();
-		for(iter=get_guide_list_y().begin();iter!=get_guide_list_y().end();++iter)
-		{
+		for (iter = get_guide_list().begin(); iter != get_guide_list().end(); ++iter){
 			if(!data.empty())
 				data+=' ';
-			data+=strprintf("%f",*iter);
+			data+=strprintf(" %f*%f*%f", (*iter).point[0], (*iter).point[1], (*iter).angle.get());
```

Comment:
1. `(*iter).` can be written as `iter->`
2. if you don't use the iterator itself, but only the value it refers to, you can make a range-based for.
`for (const auto& guide : get_guide_list) {}`

Reply:
done